### PR TITLE
Add the missing bands for the geomedian product.

### PR DESCRIPTION
**Pull request by tebadi**

### CODE REVIEW

File: products/ga_s2_gm.odc-product.yaml [hunk 1]
```diff
@@ -75,3 +75,28 @@ measurements:
   units: '1'
   dtype: uint16
   nodata: 0
+
+- name: 'SMAD'
+  aliases: [smad, sdev, SDEV]
+  units: '1'
+  dtype: uint16
```
Comment (Kirill888) [inline products/ga_s2_gm.odc-product.yaml:82]: All *MAD bands are `float32` and do not have `nodata` as they use `NaN` to encode missing data.

There is still some confusion as to what `nodata` should be when describing floating point data.

Options are:

```yaml
    nodata: null
    nodata: .nan
```

This impacts data loading and fusing, errors in configuration are not obvious or reported until weirdness is observed during data loading, and only the regions of data overlap/missing data. Not sure what products we can use as reference for how floating point data is described.

File: products/ga_s2_gm.odc-product.yaml [hunk 2]
```diff
@@ -9,8 +9,8 @@ metadata:
 storage:
   crs: EPSG:6933
   resolution:
-    x: 10
-    y: -10
+    x: 20
```
Comment (Kirill888) [inline products/ga_s2_gm.odc-product.yaml:12]: This is not right, resolution is 10m

File: products/ga_s2_gm.odc-product.yaml (hunk 1 above)
Comment (tebadi) [reply products/ga_s2_gm.odc-product.yaml:82]: `ga_ls8c_wofs_2_annual_summary.yaml` product uses `nodata: .na`

File: products/ga_s2_gm.odc-product.yaml (hunk 2 above)
Comment (tebadi) [reply products/ga_s2_gm.odc-product.yaml]: Changed this back to 10m.

File: products/ga_s2_gm.odc-product.yaml (hunk 1 above)
Comment (tebadi) [reply products/ga_s2_gm.odc-product.yaml]: `ga_ls8c_ard_3` also uses `nodata: .nan`

**Review by Kirill888 — APPROVED** [review]
thanks

**Review by omad — APPROVED** [review]
:+1:

**Pull request merged**